ipn/ipnlocal/c2n.go

```diff
@@ -175,9 +176,22 @@ func handleC2NDebugLogHeap(b *LocalBackend, w http.ResponseWriter, r *http.Reque
 		http.Error(w, "not implemented", http.StatusNotImplemented)
 		return
 	}
+	runtime.GC()
```

Comment:
unrelated to this PR.

also, I'd sent https://github.com/tailscale/tailscale/pull/11479 to make us match the Go option to do this if requested.

Reply:
Removed